pubspec.yaml

```diff
@@ -6,3 +6,5 @@ environment:
 
 dev_dependencies:
   melos: ^6.1.0
+dependencies:
+  geolocator: ^13.0.1
```

Comment:
First, why?
Second, wrong pubspec.

Reply:
The # of conflicts I'm gonna have when I push...

P.S. (You removed it, but didn't add it to the other one)

Reply:
It's in the pubspec.yaml in the app folder with the other dependencies.
![image](https://github.com/user-attachments/assets/18107358-a9b3-4f1f-989e-730fe1a2cc69)

Reply:
Huh. Ok then.